docs/sources/alerting/unified-alerting/notifications/mute-timings.md

```diff
@@ -0,0 +1,63 @@
+description = "Mute timings"
+keywords = ["grafana", "alerting", "guide", "mute", "mute timings", "mute time interval"]
+weight = 450
++++
+
+# Mute timings
+
+A mute timing is a named recurring interval of time to mute notifications for a notification policy. This is useful for preventing alert notifications from a notification policy for a period of time that reoccurs, such as a regular maintenance period. If an alert is triggered and it matches the interval then a notification will not be sent for that policy.
+
+As with silences, mute timings do not prevent alert rules from being evaluated. They also do not stop alert instances being shown in the user interface. Mute timings only prevent notifications from being created.
+
+You can configure Grafana managed mute timings as well as mute timings for an [external Alertmanager data source]({{< relref "../../datasources/alertmanager.md" >}}). For more information, see [Alertmanager]({{< relref "./fundamentals/alertmanager.md" >}}).
+
+## Mute timings vs silences
+
+While similar in effect there are a few key differences between mute timings and silences.
+
+| Mute timing                                        | Silence                                                                      |
+| -------------------------------------------------- | ---------------------------------------------------------------------------- |
+| Uses time interval definitions that can reoccur    | Has a fixed start and end time                                               |
+| Is created and then added to notification policies | Uses labels to match against an alert to determine whether to silence or not |
+
+## Create a mute timing
+
+1. In the Grafana menu, click the **Alerting** (bell) icon to open the Alerting page listing existing alerts.
+1. Click **Notification policies**.
+1. From the **Alertmanager** dropdown, select an external Alertmanager. By default, the Grafana Alertmanager is selected.
+1. At the bottom of the page there will be a section titled **Mute timings**. Click the **Add mute timing** button.
+1. You will be redirected to a form to create a [time interval](#time-intervals) to match against for your mute timing.
+1. Click **Submit** to create the mute timing.
+
+## Add mute timing to a notification policy
+
+After a mute timing is created, you need to select the notification policies to add it to.
+
+1. Identify the notification policy you would like to add the mute timing to and click the **Edit** button for that policy.
+1. From the Mute Timings dropdown select the mute timings you would like to add to the route.
+1. Click the **Save policy** button to save.
+
+## Time intervals
+
+A time interval is a definition for a moment in time. If an alert fires during this interval it will be suppressed. All fields are lists, and at least one list element must be satisfied to match the field. Fields also support ranges using `:` (ex: `monday:thursday`). The fields available for a time interval are:
```

Comment:
```suggestion
A time interval is a definition for a moment in time. If an alert fires during this interval it will be suppressed. All fields are lists, and at least one list element must be satisfied to match the field. Fields also support ranges using `:` (ex: `monday:thursday`). The fields available for a time interval are: mute timing can contain multiple time intervals. A time interval is a specific duration when alerts are suppressed from firing. The duration typically consists of a specific time range along with days of a week, month, or year. 

	All properties for the time interval are lists, and at least one list element must be satisfied to match the field. The fields support ranges using `:` (ex: `monday:thursday`). If you leave a field blank, it will match with any moment of time. 

Supported time interval options are:
```